packages/lib-utils/src/app/api-discovery/index.ts

```diff
@@ -2,17 +2,15 @@ import type { AnyObject } from '@openshift/dynamic-plugin-sdk';
 import { consoleLogger } from '@openshift/dynamic-plugin-sdk';
 import * as _ from 'lodash-es';
 import { plural } from 'pluralize';
-import type { Dispatch } from 'redux';
 import { kindToAbbr } from '../../k8s/k8s-resource';
 import type {
+  ApiActions,
```

Comment:
Nit: abbreviations should be uppercase in type names, e.g. `APIActions` instead of `ApiActions`.